creator-node/src/snapbackSM.js

```diff
@@ -0,0 +1,93 @@
+const config = require('./config')
+const Bull = require('bull')
+const utils = require('./utils')
+const { logger } = require('./logging')
+
+const devDelayInMS = 10000
+
+// Snap back state machine
+// Ensures file availability through sync and user replica operations
+class SnapbackSM {
+  constructor (audiusLibs) {
+    this.audiusLibs = audiusLibs
+    this.initialized = false
+    // Toggle to switch logs
+    this.debug = true
+    this.log(`Constructed snapback!`)
```

Comment:
nit but, this log statement should be printed after line 21 once construction has succeeded?